node/lib/openshift-origin-node/model/v2_cart_model.rb

```diff
@@ -400,6 +400,9 @@ def post_configure(cartridge_name)
           if cartridge.name == primary_cartridge.name and empty_repository?
             output << "CLIENT_MESSAGE: An empty Git repository has been created for your application.  Use 'git push' to add your code."
           else
+            if @state.value == State::STARTED
+              output << stop_cartridge(cartridge, user_initiated: true)
+            end
             output << start_cartridge('start', cartridge, user_initiated: true)
```

Comment:
Why are we stopping the cartridge above? Why not just throw the inverse of that if statement around this line, so that the cartridge stays started and doesn't get re-started? Seems silly to start-stop-start when we could just start once.